spec/features/login_spec.rb

```diff
@@ -3,6 +3,12 @@
 require "rails_helper"
 
 RSpec.describe "Login in" do
+  # rubocop:disable RSpec/AnyInstance
+  before do
+    allow_any_instance_of(ApplicationHelper).to receive(:in_staff_subnet?).and_return(false)
+  end
+  # rubocop:enable RSpec/AnyInstance
+
```

Comment:
Is this is non-staff attempting login (since returning false)? Name of test seems incomplete?

Reply:
Yeah, there's two other specs for staff login, one inside the subnet (`spec/features/staff_login_in_subnet_spec.rb`) and one outside the subnet (`spec/features/staff_login_outside_subnet_spec.rb`).